deployments/clowdapp.yml

```diff
@@ -120,10 +120,34 @@ objects:
       schedule: ${CLEANER_SCHEDULE}
       suspend: ${{CLEANER_SUSPEND}}
       podSpec:
-        image: quay.io/cloudservices/payload-tracker-go:${IMAGE_TAG}
+        image: quay.io/cloudservices/postgresql-rds:12
         restartPolicy: Never
-        command:
-          - ./tools/cron-job.sh
+        volumes:
+          - name: playbook-tracker-go-db-cleaner-volume
+            configMap:
+              name: playbook-tracker-go-db-cleaner-config
+        volumeMounts:
+          - name: playbook-tracker-go-db-cleaner-volume
+            mountPath: /playbook-tracker-go-db-cleaner
+        command: [ 'sh', '/playbook-tracker-go-db-cleaner/clean.sh' ]
+        env:
+          - name: PGHOST
+            valueFrom:
+              secretKeyRef:
+                name: payload-tracker-db-creds
```

Comment:
This secret is taken from [our config](https://gitlab.cee.redhat.com/service/app-interface/-/blob/master/data/services/insights/payload-tracker/namespaces/payload-tracker-prod.yml#L72).